### PR TITLE
chore(build): Add dry run option

### PR DESCRIPTION
**Pull request by mickr**
Adding dry_run option to release script for easier testing.

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @mickr has signed the CLA. Thanks for the pull request!

File: build/release.sh [hunk 1]
```diff
@@ -6,6 +6,19 @@ export NODE_PATH=$NODE_PATH:./node_modules
 OLD_VERSION="XXX"
 VERSION="XXX"
 
+dry_run=false
```
Comment (jstoffan) [inline build/release.sh]: It looks like our current variables follow SNAKE_CAPS convention.

**Comment by jstoffan** [comment]
I think the scope should probably be `chore` rather than `feat` for this commit, since the change doesn't affect the end user.

File: build/release.sh (hunk 1 above)
Comment (mickr) [reply build/release.sh]: Most of the variables do, our switches do not. Since this was a command line switch I made it look like the others

**Pull request merged**